Tasks/Common/Sanitizer/ArgumentsSanitizer.ps1

```diff
@@ -0,0 +1,56 @@
+$featureFlags = @{
+    activate  = [System.Convert]::ToBoolean($env:AZP_MSRC75787_ENABLE_NEW_LOGIC)
```

Comment:
This method will fail when env variable is defined as empty. Null coalescing is recommended.